diff --git a/camelot/io.py b/camelot/io.py
--- a/camelot/io.py
+++ b/camelot/io.py
@@ -16,12 +16,13 @@
 def read_pdf(
     filepath,
     pages="1",
     password=None,
     flavor="lattice",
     suppress_stdout=False,
-    parallel=False,
+    parallel=False,
+    workers=None,
     layout_kwargs=None,
     debug=False,
     **kwargs,
 ):
@@ -66,7 +67,14 @@
     if layout_kwargs is None:
         layout_kwargs = {}
 
     handler = PDFHandler(filepath, pages=pages, password=password, debug=debug)
-    tables = handler.parse(flavor=flavor, suppress_stdout=suppress_stdout, parallel=parallel, layout_kwargs=layout_kwargs, **kwargs)
+    tables = handler.parse(
+        flavor=flavor,
+        suppress_stdout=suppress_stdout,
+        parallel=parallel,
+        workers=workers,
+        layout_kwargs=layout_kwargs,
+        **kwargs,
+    )
     return remove_extra(tables)
